ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -539,35 +539,40 @@ class PackedBubbleSeries extends BubbleSeries {
 
         let parentAttribs: SVGAttributes = {};
 
-        // create the group for parent Nodes if doesn't exist
-        if (!this.parentNodesGroup) {
-            this.parentNodesGroup = this.plotGroup(
-                'parentNodesGroup',
-                'parentNode',
-                this.visible ? 'inherit' : 'hidden',
-                0.1, chart.seriesGroup
-            );
-            (this.group as any).attr({
-                zIndex: 2
-            });
-        }
+        // Create the group for parent Nodes if doesn't exist
+        // If exists it will only be adjusted to the updated plot size (#12063)
+        this.parentNodesGroup = this.plotGroup(
+            'parentNodesGroup',
+            'parentNode',
+            this.visible ? 'inherit' : 'hidden',
+            0.1, chart.seriesGroup
+        );
+        this.group?.attr({
+            zIndex: 2
+        });
 
         this.calculateParentRadius();
-        parentAttribs = merge({
-            x: (this.parentNode as any).plotX -
-                (this.parentNodeRadius as any),
-            y: (this.parentNode as any).plotY -
-                (this.parentNodeRadius as any),
-            width: (this.parentNodeRadius as any) * 2,
-            height: (this.parentNodeRadius as any) * 2
-        }, parentOptions);
-        if (!(this.parentNode as any).graphic) {
-            this.graph = (this.parentNode as any).graphic =
-                chart.renderer.symbol((parentOptions as any).symbol)
-                    .add(this.parentNodesGroup);
+        if (
+            this.parentNode &&
+            this.parentNode.plotX &&
+            this.parentNode.plotY &&
```

Comment:
When checking the number-type value, always use `H.defined()`, so `0`-values are correctly handled. For example `plotX = 0` and `plotY = 0` (top left corner) would be skipped forever if that happens (unlikely, but still)